output/mistral_fluentbit/readme.rst

```diff
@@ -0,0 +1,44 @@
+Mistral Fluent Bit plug-in
+==========================
+
+This plug-in receives violation data from Mistral and sends it to Fluent Bit.
```

Comment:
Might be worth stating here how data is sent to Fluent Bit - as there are quite a few options.

Reply:
I have explained what option we use to connect to Fluent Bit.